lib/ci-helper.ts

```diff
@@ -511,19 +511,24 @@ export class CIHelper {
                     comment.author} is not permitted to use GitGitGadget`);
             }
 
+            const pr = await this.github.getPRInfo(comment.prNumber);
```

Comment:
Hmm. This will add a roundtrip even if we do not need the information. Maybe we can just repeat that one line below, in only the places where we actually need `pr`?

Reply:
Done.

Reply:
Excellent!